src/components/PassphraseForm.jsx

```diff
@@ -87,9 +91,18 @@ class PassphraseForm extends Component {
       location.search
     )
 
+    const canAuthWithPassword = get(
+      client,
+      'capabilities.can_auth_with_password'
+    )
+    const canAuthWithOIDC = get(client, 'capabilities.can_auth_with_oidc')
+    const isOIDC = !canAuthWithPassword && canAuthWithOIDC
```

Comment:
can't we have both? I mean having a canAuthWithOIDC but also having a password?

Reply:
I think we can have both, and if we have both we do not want to have the OIDC title.

Reply:
OK thanks for the explanation. So a small nit: `shouldUseOIDCtitle` : ... :D